dashboard/app/dsl/script_dsl.rb

```diff
@@ -259,7 +259,7 @@ def self.serialize_level(
     l = "#{type} '#{level.key.gsub("'") {"\\'"}}'"
     l += ', active: false' if experiments.empty? && active == false
     l += ', active: true' if experiments.any? && (active == true || active.nil?)
-    l += ", experiments: ['#{experiments.join(', ')}']" if experiments.any?
+    l += ", experiments: [#{experiments.map {|e| "'#{e}'"}.join(', ')}]" if experiments.any?
```

Comment:
That's slightly different, but close enough (double quotes instead of single, no space between elements). Done.